sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -350,25 +352,43 @@ private[parquet] class ParquetFilters(
   }
 
   /**
-   * Returns a map from name of the column to the data type, if predicate push down applies.
+   * Returns nameMap and typeMap based on different case sensitive mode, if predicate push
```

Comment:
instead of returning 2 maps, can we just add a `originalName` field to `ParquetSchemaType`?

Reply:
+1 for avoiding returning 2maps if possible.

Reply:
Great idea!